### DISCUSSION TITLE
Some examples are missing in the homepage links array

### DISCUSSION THREAD

**Discussion by cloud-walker** [ariakit/ariakit#1751] [General]
I've noticed that from my recent PR #1732 and also from older ones (like #1219), the examples are not automatically linked to the examples page (currently the homepage).

If appreciated I could work on to link them as I find them, or even better automatically load them.

Let me know 😄

**Comment by diegohaz** [comment]
Yeah! That's temporary. I'm already working on new docs that will list the pages automatically. I should land an initial version next week.

**Reply by cloud-walker** [reply]
so, better to wait your work?
